std/algorithm/setops.d

```diff
@@ -888,39 +1013,81 @@ struct MultiwayMerge(alias less, RangeOfRanges)
 }
 
 /// Ditto
-MultiwayMerge!(less, RangeOfRanges) multiwayMerge
+auto multiwayMerge
 (alias less = "a < b", RangeOfRanges)
 (RangeOfRanges ror)
+if (isSortedRange!(ElementType!(RangeOfRanges)))
 {
-    return typeof(return)(ror);
+    return multiwayMergeImpl!less(ror);
+}
+
+/// Ditto
+//deprecated("The use with non `SortedRange` ranges is deprecated. Please use sort or assumeSorted.")
```

Comment:
Why are these commented out?

Reply:
If you  un-comment them then Phobos doesn't compile due to deprecation settings. Maybe `pragma(msg)` could be used here?